### ISSUE TITLE
Duet detection seems to fail for relative songs

### ISSUE THREAD

**Issue by bohning** [bohning/usdb_syncer#43]
If a song has relative timings (#RELATIVE:yes), it seems to be a falsely detected as duet.

**Comment by RumovZ** [comment]
Do you have an example for a relative song? How do actual duets work with #RELATIVE? 
On a related note, should we write relative songs at all or convert them to absolute ones?

**Comment by bohning** [comment]
I manually converted all relative songs in the usdb to absolute timings a while ago, so only one recently uploaded song had relative timings (not any longer though, it now has absolute timings). I believe, but am not positive on this, that duets are not possible with relative timings (I can’t remember having seen a duet with relative timings in all those years).

Relative timings are somewhat obsolete/unused, so I would suggest to convert them to absolute timings anyways. However, it is pretty easy to save a song with relative timings - simply save the song in the USDX editor with SHIFT+S instead of S and the resulting file will have relative timings (little known shortcut I guess).

I added it to the longer list of potential cleanup tasks in #20 a while ago.
